### DISCUSSION TITLE
How to actually use this plugin ?

### DISCUSSION THREAD

**Discussion by unmeshvrije** [Vinzent03/obsidian-git#100] [Q&A]
In the `How to use` section following features are listed.
But how can I use them ?
```
 
Features
Automatic vault backup every X minutes
Pull changes from remote repository on Obsidian startup
Assign hotkeys for pulling/pushing changes to a remote repository
```

I have setup the Github repository and can push the `md` files from my local Obsidian vault.
I followed the manual installation step and extracted the tar file in my `<VAULT>/.obsidian/plugins`
but I am lost on what to next in order to enjoy the aforementioned features.

any pointers would be greatly appreciated,
Unmesh

**Comment by Vinzent03** [comment]
You don't need `Source code (tar.gz)`, you need the `obsidian-git-1.9.2.zip` file. But the easiest way to install it is via the plugin store in Obsidian.

**Comment by unmeshvrije** [comment]
I have Obsidian `0.12.4` but there is no `Third Party plugins` option. 
Existing options (core plugins, community plugins) do not have the `Obsidian Git` listed . Thus, I have to resort to manual installation.

Now I have unzipped the `obsidian-git-1.9.2.zip` in the `plugins` directory of my vault. 
However, I am still lost as to
1. How would this backup automatically every X minutes 
2. pull changes from remote repository on Obsidian startup 
3. ...

All other features that I am expecting.
Sorry if this question sounds naive, but I am lost.

**Comment by Vinzent03** [comment]
@unmeshvrije They changed the name. `Third Party plugins` is now `community plugins`. Just search for `git` and install and **enable** the plugin.

**Comment by Chocrates** [comment]
@unmeshvrije I was having trouble getting it to actually push as well.  Turns out it by default it doesn't push every x minutes.
If you go to settings you can see community plugin settings on the bottom and obsidian-git has some.

This issue has some screenshots that can help https://github.com/denolehov/obsidian-git/issues/39#issuecomment-780455129

**Comment by Vinzent03** [comment]
@unmeshvrije Does the problem still persist?

**Comment by skullbunnygalaxy** [comment]
I had this same issue as a new user of Obsidian with this being the first plugin I've installed. It would be helpful to add to the Installation page on the wiki where to go to configure settings for the plugin. I was confused by it not being listed on the plugin install page.

**Reply by Vinzent03** [reply]
What exactly do you have in mind? What settings are not self explained enough?

**Reply by skullbunnygalaxy** [reply]
Underneath either the installation directions or requirements, I would add a subheading like this:

### Configuration
After Installation, go to Obsidian's Settings Menu, scroll down to view Plugin Options, and select Obsidian Git. Here you can configure the plugin settings including vault backup interval, auto pull interval, commit message, set branch, and more.

**Reply by sourcecodemage** [reply]
also recommended settings that work for most people. I'm stuck on trying to figure out what put for interval settings for backup and pull , since I want to sync multiple devices.  I have no idea what to put in the interval fields.  I def don't want to use settings that will cause merge conflicts!
